src/main/java/solrocr/OcrHighlighter.java

```diff
@@ -435,10 +435,18 @@ public OcrHighlightResult[] highlightOcrFields(
             // Stop highlighting
             break docLoop;
           } catch (RuntimeException e) {
-            // This catch-all prevents OCR highlighting from failing the complete query, instead
-            // users
-            // get an error message in their Solr log.
-            log.error("Could not highlight OCR content for document", e);
+            // This catch-all prevents OCR highlighting from failing the complete query, 
+            // instead users get an error message in their Solr log.
+            String sourcePtr = "n.a.";
+            if(content.getPointer() != null && content.getPointer().sources != null) {
+              List<String> pathStrs = content.getPointer().sources.stream()
+                .map(fileSource -> fileSource.path.toString()).collect(Collectors.toList());
+              sourcePtr = String.join(",", pathStrs);
```

Comment:
Why not log the whole pointer? I think the offsets are invaluable for debugging if you're using them.

Reply:
But how to log the pointer in a meaningful way? 
The clazz itself contains no human-readable `toString`-implementation.

Reply:
Ouch, you're right, my bad! That should then be a part of this change set, basically the inverse of `SourcePointer.parse(String ptr)`:

```java
  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    for (FileSource source : this.sources) {
      if (sb.length() > 0) {
        sb.append("+");
      }
      sb.append(source.path);
      if (source.isAscii) {
        sb.append("{ascii}");
      }
      if (!source.regions.isEmpty()) {
        sb.append("[");
        for (Region region : source.regions) {
          if (sb.charAt(sb.length() - 1) != '[') {
            sb.append(",");
          }
          sb.append(region.start);
          sb.append(":");
          sb.append(region.end);
        }
        sb.append("]");
      }
    }
    return sb.toString();
  }
```